repository/src/main/java/org/alfresco/repo/search/impl/DbOrIndexSwitchingQueryLanguage.java

```diff
@@ -125,15 +133,15 @@ public ResultSet executeQuery(SearchParameters searchParameters)
             {
                 if(logger.isDebugEnabled())
                 {
-                    logger.debug("Using SOLR query: "+dbQueryLanguage.getName()+" for "+searchParameters);
+                    logger.debug("Using "+subsystemName+" query: "+dbQueryLanguage.getName()+" for "+searchParameters);

```

Comment:
Just repeating myself from somewhere else, because I don't see the comment on this PR.  We could remove the if statement by using slf4j's token syntax:
```
LOGGER.debug("Using {} query: {} for {}", subsystemName, dbQueryLanguage.getName(), searchParameters);
```

Reply:
FYI: it looks as if we have at least two logging facades (and code style) across the codebases. Majority is still org.apache.commons.logging and some (approx 77 files) is now org.slf4j.